views/premium.html

```diff
@@ -184,6 +184,32 @@ <h1><%= _x('100% Satisfaction Guarantee', 'This text resides in the Premium page
     </p>
   </div>
   <hr>
+  <div class="mailpoet-premium-page-text">
```

Comment:
According to the screenshot on specs, this should go before the "Try MailPoet Premium for 30 days" section.

Reply:
Fixed.